commands/01-mod.js

```diff
@@ -158,18 +158,37 @@ const Module = new Augur.Module()
           if (bannerHighRole.comparePositionTo(bannedHighRole) <= 0) {
             msg.reply(`you cannot ban ${u.escapeText(member.displayName)}!`);
             continue;
-          } else {
+          }
+          members.set(userId, member);
+        }
+        else {
+          members.set(userId, undefined);
+        }
+      }
```

Comment:
Close the `try/catch` block
```suggestion
      } catch(error) { u.errorHandler(error, msg); }
```